addons/tutorial_01_get_started/manifest.json

```diff
@@ -26,39 +27,39 @@
         }],
         "next": {
           "op": "signal",
-          "qml_emitter": "serverListButton-btn",
+          "qml_emitter": "//serverListButton-btn",
           "signal": "visibleChanged"
         }
       },
       {
         "id": "countryAU",
-        "element": "serverCountryList/serverCountry-au",
```

Comment:
the point here is to have a way to pick the first country without specifying its code/city. I dont want to change the IDs because I want to avoid the retranslation of the strings.